README.md

```diff
@@ -150,7 +150,7 @@ CMake option                  | Description
 `FIRESTARTER_BUILD_TYPE`      | Can be any of `FIRESTARTER`, `FIRESTARTER_CUDA`, `FIRESTARTER_ONEAPI`, or `FIRESTARTER_HIP`. Default `FIRESTARTER`
 `FIRESTARTER_LINK_STATIC`     | Link FIRESTARTER as a static binary. Note, dlopen is not supported in static binaries. This option is not available on macOS or with CUDA or OneAPI enabled. Default `ON`
 `FIRESTARTER_BUILD_HWLOC`     | Build hwloc dependency. Default `ON`
-`FIRESTARTER_THREAD_AFFINITY` | Enable FIRESTARTER to set affinity to hardware threads. Default `ON`
+`FIRESTARTER_FETCH_GOOGLETEST`| Fetch the GoogleTest dependency. Default: `ON`
```

Comment:
Can we change the default to OFF?

Reply:
If we change the default to OFF, we will depend on Google Test being already installed on the system. I don't think that we can assume this as a given. However are able to disable UnitTests via a toggle, which in turn would allow us to guards the build dependency on GoogleTest.
In this case I would suggest adding a toggle `FIRESTARTER_BUILD_UNITTESTS`.

Reply:
`FIRESTARTER_BUILD_UNITTESTS` sounds good